Gruntfile.coffee

```diff
@@ -301,10 +301,10 @@ module.exports = (grunt) ->
 
 	grunt.util.linefeed = "\n"
 	# Project configuration.
-	grunt.initConfig
+	@initConfig
 
 		# Metadata.
-		pkg: grunt.file.readJSON("package.json")
+		pkg: @file.readJSON("package.json")
```

Comment:
If you're going to change `grunt` to `@`, it should probably be done everywhere. 3 lines below is the same thing, but not changed